### ISSUE TITLE
Motto Or Tagline Change

### ISSUE THREAD

**Issue by AllanRegush** [EddieJaoudeCommunity/EddieJaoudeCommunity.github.io#115]
Eddie had thought of "Opensource is in our DNA"  Should we change our Motto or Tagline?

**Comment by eddiejaoude** [comment]
Thanks @AllanRegush 👍

**Comment by BOLT04** [comment]
I like that idea 😄!!
Maybe instead of "We believe Open Source is for everyone, yes you! OPEN SOURCE is NOT about code, it is about people and collaboration ", we can have "Opensource is in our DNA. We believe everyone can contribute to Open Source and collaborate with the community!"

**Comment by VictoryWekwa** [comment]
Hello, can i work on this issue please?

**Comment by eddiejaoude** [comment]
Sure @VictoryWekwa 👍

**Comment by Vyvy-vi** [comment]
> We believe Open Source is for everyone, yes you! 
This is quite catchy though^

**Comment by eddiejaoude** [comment]
We could use both in different places - one at the top and one at the bottom of the page 🤓

**Comment by eddiejaoude** [comment]
We can keep tweaking it 😄

**Comment by BOLT04** [comment]
I agree with @eddiejaoude on one on the top and one at the bottom of the page, since having a CTA on the top and the bottom is common (I think), so that text would go along that 👍

**Comment by The-Phantom-666** [comment]
What about "Open Source is the lifeblood of our community"

**Comment by eddiejaoude** [comment]
btw I unassigned this issue as it was being blocked. I think we should update the website and keep tweaking it with new ideas in new issues

Thanks @The-Phantom-666 , not sure I like `lifeblood` though - I don't know why, just sounds a bit bad 🤷‍♂️

**Comment by MehulKChaudhari** [comment]
I got this Idea "OpenSourse is the way of life"

**Comment by dumbcoder7** [comment]
~~This should be a discussion~~

**Comment by Vyvy-vi** [comment]
I think there was a discussion opened for this 🤔

**Comment by MehulKChaudhari** [comment]
oops, I wasn't meant to do that ??  @Vyvy-vi

**Comment by eddiejaoude** [comment]
Let's add value quickly by changing to the initial ones at the top of this issue that we agreed, then later it can be tweaked further

**Comment by Skyhero-admin** [comment]
Is this discussion still open?
If it is then can we keep it what Eddie always says "Collaboration is First and Code is Second "

**Comment by adityaraute** [comment]
Love this one

**Comment by Vyvy-vi** [comment]
Or one up - We can have the motto on the site change :wink:

**Comment by BOLT04** [comment]
@Skyhero-admin that is a good idea too 👍. I think the PR that gets opened first might be the merged one, since it already adds value 😄

**Comment by divyakumari12** [comment]
This tagline is looking good "Opensource is in our DNA" .
It's perfect.😃

**Comment by eddiejaoude** [comment]
Lets do it 👍 

@Vyvy-vi idea of having it change is a great idea too - but this can come later

**Comment by adityaraute** [comment]
So which one then? let's decide. This issue been here for 5 months now lol

**Comment by eddiejaoude** [comment]
We could have it change randomly for each refresh 🤔

**Comment by MeghaKeshri** [comment]
well what is the current tagline

**Comment by eddiejaoude** [comment]
Our main one is "Collaboration 1st Code 2nd" 🔥

**Comment by adityaraute** [comment]
I second this
